Fix and improve filter choices (#3812) (#3811)
GeotrekCE/Geotrek-admin: docs/changelog.rst

2.101.3+dev (XXXX-XX-XX)
------------------------

**Bug fixes**

> - Fix filters tigger exception with data and not updated until application restart (#3812)

Author: ```suggestion
- Fix: filters choices can raise exception in lists and not updated until application restart (#3812)
```
